lib/loader.js

```diff
@@ -63,7 +63,7 @@ function sassLoader(content) {
     ));
 
     // Skip empty files, otherwise it will stop webpack, see issue #21
-    if (options.data.trim() === "") {
+    if (typeof options.data !== "string" || options.data.trim() === "") {
```

Comment:
Let's add additional comment about why this and we can merge this :+1: (don't remove current comment just add new below)